dds_cli/account_manager.py

```diff
@@ -9,6 +9,7 @@
 
 # Installed
 import http
+from pydoc import resolve
```

Comment:
I think this was added automatically by VS Code?

Reply:
Ah, yeah it was.